apps/src/applab/designMode.js

```diff
@@ -188,7 +203,7 @@ designMode.onPropertyChange = function(element, name, value) {
     default:
       throw "unknown property name " + name;
   }
-  Applab.levelHtml = designMode.serializeToLevelHtml();
+  designMode.serializeToLevelHtml();
```

Comment:
This was the case before as well. Can you think of any reason we shouldn't instead just do this when we toggle between code/design mode or hit run? (And probably eventually if we have autosave, we'll want to do it then too)